content/docs/studio/get-started.md

```diff
@@ -26,10 +32,22 @@ more views._
 4. To create a new view, click on `Add a View`. All the organizations that you
    have access to will be listed.
 
-> To create views from your GitHub repositories, you must install the Iterative
-> Studio GitHub app. Refer to the section on
-> [GitHub app installation](/doc/studio/user-guide/install-github-app) for more
-> details.
+<admon type="info">
+
+If you do not see your desired organizations or Git repositories, make sure that
```

Comment:
💅🏼  This whole block should be indented 2 spaces so it's part of the list item.

Reply:
Done.